doc/release-notes.md

```diff
@@ -347,6 +347,16 @@ RPC
   - Fee estimation failed
   - Transaction has too long of a mempool chain
 
+- The `sendrawtransaction` error code for exceeding `maxfeerate` has been changed from
+  `-26` to `-25`. The error string has been changed from "absurdly-high-fee" to
```

Comment:
I think this should also list the name for these error codes as the number itself is a harder to undertsand.

Reply:
We can adjust this in the wiki closer to release.